rfc/src/rfcs/0012-list.md

```diff
@@ -132,6 +152,30 @@ For example:
 }
 ```
 
+#### Output Schema
+The formal output schema is:
+```
+Kani Version: float
+File Version: int
```

Comment:
Nit: I'm assuming you mean JSON version?

Reply:
No, see https://github.com/model-checking/kani/pull/3463#discussion_r1739455268.

Reply:
In that case, can you clarify in the RFC what version/format of JSON are you using in your output?

Reply:
Are you suggesting that we also include the JSON format?

Reply:
Yes, but since we anticipate this to be a mostly constant value, I just thought we should document this in the RFC itself, and then we dont have to include the jSON version in the JSON response object itself.

Reply:
I feel like if the JSON version subject to change in future versions of Kani there's not much point putting it in the RFC? I'd prefer to only put information in the RFC that we expect to be (relatively) stable--we'll almost certainly update the JSON version over time.